test/e2e/framework/volume_util.go

```diff
@@ -53,29 +53,37 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
 )
 
 const (
+  // Kb is byte size of kilobyte
```

Comment:
The indent of this comment seems different from the actual code.
Is that just due to the web browser view? or actually different on the code?

Reply:
I'll fix this.